src/control/utils.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+/* eslint-disable @typescript-eslint/naming-convention */
+import { estypes } from '@elastic/elasticsearch';
+import { CommonRequestParameters, IApplication, IConfig } from '../common/interfaces';
+import { elasticConfigPath } from '../common/constants';
+import { ElasticDbClientsConfig } from '../common/elastic/interfaces';
+import { Item } from '../control/item/models/item';
+import { Tile } from '../control/tile/models/tile';
+import { Route } from '../control/route/models/route';
+import { ConvertSnakeToCamelCase } from '../common/utils';
+import { CONTROL_FIELDS } from './constants';
+import { ControlResponse } from './interfaces';
+
+const LAST_ELEMENT_INDEX = -1;
+
+const generateDisplayName = <T extends Tile | Item | Route>(
```

Comment:
Please create an `enum` for the common used `strings` representing types and other entities.

Reply:
I would like you to elaborate :)